<launch>

    <!-- Node for sending images from bag -->
	<arg name="video_folder_prefix" default="" />
	<node name="extract_bag_depth" pkg="franka_irom" type="extract_bag_depth.py" output="screen" args="$(arg video_folder_prefix)"/>

	<!-- Depth to point cloud conversion -->
    <!-- http://wiki.ros.org/depth_image_proc#depth_image_proc.2BAC8-point_cloud_xyz -->
    <!-- <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
    <node pkg="nodelet" type="nodelet" name="cloudify"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
        <remap from="image_rect" to="/depth/image_raw"/>
        <remap from="camera_info" to="/depth/camera_info"/>
        <remap from="points" to="cloudify/points"/>
    </node> -->

</launch>